README.md

```diff
@@ -1,7 +1,6 @@
 # Open Neural Network eXchange (ONNX) Model Zoo
-
-[![Generic badge](https://img.shields.io/badge/Status-Work_In_Progress-red.svg)](#) 
 [![Generic badge](https://img.shields.io/badge/Contributions-Welcome-brightgreen.svg)](contribute.md)
+[![Generic badge](https://img.shields.io/badge/Status-Work_In_Progress-red.svg)](#) 
```

Comment:
Everything is always "in progress". Suggest removing this.

Reply:
Removed!